lib/appsignal/integrations/excon.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Appsignal
+  module Integrations
+    module ExconIntegration
+      def self.instrument(name, data, &block)
+        namespace, *event = name.split(".")
+        rails_name = [event, namespace].flatten.join(".")
+
+        title =
+          if rails_name == "response.excon"
+            data[:host]
+          else
+            "#{data[:method].to_s.upcase} #{data[:scheme]}://#{data[:host]}"
```

Comment:
I added a fix here by calling `to_s` on `data[:method]`. It would return a Symbol instead of a String in a test app.